src/main/resources/XWiki/Macros/TeamConfiguration.xml

```diff
@@ -0,0 +1,420 @@
+<?xml version='1.1' encoding='UTF-8'?>
+<xwikidoc version="1.4" reference="XWiki.Macros.TeamConfiguration" locale="">
```

Comment:
You need to make this page editable in the POM, because by default all pages that are part of a paid (licensed) extension are protected. Otherwise the users won't be able to use the administration section. See https://github.com/xwikisas/application-diagram/blob/master/application-diagram-ui/pom.xml#L43-L47 .